### PR TITLE
feat(docs): add the a11y section for `CloseButton` component

### LINKED ISSUE #1563: Add accessibility section for ` close-button`
### Subject

accessibility section for `close-button`

#### Create an accessibility section using the link below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes
https://mantine.dev/core/close-button/#accessibility
#### Include the following content
* aria, tabindex property is suported

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by koralle**
Closes #1563

### in English

image

### in Japanese

image

### CODE REVIEW

File: docs/contents/components/other/close-button/accessibility.ja.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: CloseButton
+description: "CloseButtonは、基本的にコンポーネントの閉じる機能をトリガーするために使用するコンポーネントです。"
+package-name: "@yamada-ui/close-button"
```
Comment (Tomoya-Matsubara) [inline docs/contents/components/other/close-button/accessibility.ja.mdx:4]: It seems the `order` is missing: https://github.com/yamada-ui/yamada-ui/pull/2288/commits/44f096087fd3917d479f36f6c5475a270006b33c.

Considering those of other components, I guess the appropriate order will be `3` (`1` for Props and `2` for Theming).

File: docs/contents/components/other/close-button/accessibility.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: CloseButton
+description: "CloseButton is a component used primarily to trigger the close functionality of a component."
+package-name: "@yamada-ui/close-button"
```
Comment (Tomoya-Matsubara) [inline docs/contents/components/other/close-button/accessibility.mdx:4]: Same here. The `order` is missing.

File: docs/contents/components/other/close-button/accessibility.ja.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`CloseButton`は、アクセシビリティに関して[WAI-ARIA Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/)に従います。
```
Comment (Tomoya-Matsubara) [inline docs/contents/components/other/close-button/accessibility.ja.mdx:10]: nits

I noticed that the link text is slightly different in the Japanese version and the English version.

I think it is more natural to use the same text in both languages:

```suggestion
`CloseButton`は、アクセシビリティに関して[WAI-ARIA - Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/)に従います。
```

#### Reference

Other components follow the `WAI-ARIA - XXX Pattern` style:

https://github.com/yamada-ui/yamada-ui/blob/44f096087fd3917d479f36f6c5475a270006b33c/docs/contents/components/disclosure/accordion/accessibility.ja.mdx?plain=1#L10

https://github.com/yamada-ui/yamada-ui/blob/44f096087fd3917d479f36f6c5475a270006b33c/docs/contents/components/disclosure/tabs/accessibility.ja.mdx?plain=1#L10

Comment (koralle) [reply docs/contents/components/other/close-button/accessibility.ja.mdx]: I fixed it.
Thank you.

**Review by Tomoya-Matsubara — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
@koralle @Tomoya-Matsubara 

Other files (props, theming) also require an `order`.

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**